coordinator/coordinator.go

```diff
@@ -356,7 +356,7 @@ func (c *Coordinator) runSQLTransformationJob(transformSource *metadata.SourceVa
 	return nil
 }
 
-func (c *Coordinator) runPrimaryTableJob(transformSource *metadata.SourceVariant, resID metadata.ResourceID, offlineStore provider.OfflineStore, schedule string) error {
```

Comment:
Is this schedule string field supposed to be removed in all these?

Reply:
yes, since primary table jobs never need a schedule since they're relational